helm-charts/0.24.5/charts/spire/values.yaml

```diff
@@ -78,7 +84,9 @@ autoscaling:
   # targetMemoryUtilizationPercentage: 80
 
 data:
-  persistent: false
+  persistent: true
```

Comment:
Defaulting this to `true` will mess up our testing setup.

If we are not managing the persistent volume for the user too, it’s best to leave this false.

The user can create a PV separately and can override this if they need.

Reply:
Yea, will revert this, I was set to true for helm template testing.